app/services/user_invite_service.rb

```diff
@@ -1,16 +1,16 @@
 class UserInviteService
   include ServicePattern
 
-  def initialize(user, organisation)
+  def initialize(user, organisation, service)
```

Comment:
Can we not still determine the service from the user? Possible a method on the child classes, eg:
```
class Placements::User < User
  def service
    :placements
  end
end
```

So when you are sending the emails, we don't have to think about the possibility of sending an email to a `Claims::User` with the service `placements`

Reply:
resolved in [0b42724](https://github.com/DFE-Digital/itt-mentor-services/pull/168/commits/0b42724ac7040f437f56b077f6d0a7e70a85fea9)